apps/common/contexts/useCurve.tsx

```diff
@@ -54,21 +55,27 @@ export const CurveContextApp = ({children}: {children: React.ReactElement}): Rea
 	) as unknown as SWRResponse<TDict<TCoinGeckoPrices>>;
 
 	const	{data: gaugesWrapper} = useSWR(
-		'https://api.curve.fi/api/getGauges?blockchainId=ethereum',
+		'https://api.curve.fi/api/getAllGauges?blockchainId=ethereum',
 		curveFetcher,
 		{revalidateOnFocus: false}
-	) as SWRResponse<{gauges: TDict<TCurveGauges>}>;
+	) as SWRResponse<TDict<TCurveGauges>>;
 
 	const	gauges = useMemo((): TCurveGauges[] => {
 		const	_gaugesForMainnet: TCurveGauges[] = [];
-		for (const gauge of Object.values(gaugesWrapper?.gauges || {})) {
+		for (const gauge of Object.values(gaugesWrapper || {})) {
 			const	currentGauge = gauge as TCurveGauges;
```

Comment:
I would remove this line entirely and just use `gauge` instead of `currentGauge`, it seems unnecessary to create a new constant. Also, `gauge` should already be inferred to be of type `TCurveGauges`